release_dashboard/lib/logic/error_to_string.dart

```diff
@@ -4,14 +4,14 @@
 
 import 'package:conductor_core/conductor_core.dart';
 
-/// Converts the error and stack trace objects to strings.
+/// Converts the error and stack trace objects to strings for display.
 String errorToString(Object error, StackTrace stackTrace) {
   final StringBuffer buffer = StringBuffer();
   if (error is ConductorException) {
     buffer.writeln('Conductor Exception:\n$error');
```

Comment:
You can do `buffer.writeln('${error.runtimeType}:');` and remove the conditional